packages/ui-service-inquirer/src/translations/ui-service-inquirer.i18n.json

```
@@ -0,0 +1,18 @@
+{
+    "MESSAGE_OBJECT_TYPE": "Object Type",
+    "MESSAGE_BUSINESS_OBJECT_INTERFACE": "Business Object Interface",
+    "MESSAGE_ABAP_CDS_SERVICE": "ABAP Core Data Service",
+    "PROMPT_DRAFT_ENABLED": "Draft Enabled",
+    "PROMPT_LAUNCH_APP_GEN": "Do you want to create an SAP Fiori application with the newly generated service?",
+    "PROMPT_LAUNCH_APP_GEN_BREADCRUMB": "Create SAP Fiori Application?",
+    "ERROR_VALIDATING_CONTENT": "There was a backend validation error for the service content and package provided, service generation cannot continue.",
+    "ERROR_FETCHING_CONTENT_FOR_SERVICE_BINDING": "Error fetching content for business object service binding",
+    "PROMPT_SERVICE_NAME_BREADCRUMB": "Service Name",
+    "PROMPT_SERVICE_NAME": "Service Name (not editable)",
+    "INFO_APP_GEN_LAUNCH": "The SAP Fiori generator will launch after the UI service has been created",
+    "ERROR_FETCHING_BUSINESS_OBJECTS": "Error fetching business object interfaces from backend system",
```

```suggestion
    "ERROR_FETCHING_BUSINESS_OBJECTS": "Error fetching business object interfaces from the backend system",
```
What advice can we give to users to fix this? Should they try again?